test/sandbox/sandbox.go

```diff
@@ -366,6 +372,12 @@ func (se StackEntry) RelPath() string {
 	return se.DirEntry.relpath
 }
 
+// CreateDir creates a directory inside the stack directory. The relpath must be
+// relative to the stack directory.
+func (se StackEntry) CreateDir(relpath string) DirEntry {
+	return newDirEntry(se.t, se.abspath, relpath)
+}
+
```

Comment:
Since StackEntry is composed by DirEntry:https://github.com/mineiros-io/terramate/blob/1772847e0eacb20b370ff20fb5db58a2afe167a3/test/sandbox/sandbox.go#L60

Do we need the method defined again on it ? I think syntatic sugar for composition would just call the DirEntry method.

Reply:
I forgot StackEntry just embedded DirEntry. =)